README.md

```diff
@@ -43,6 +43,13 @@ Find security vulnerabilities, compliance issues, and infrastructure misconfigur
 <img alt="Cloud Deployment Kit" src="docs/img/logo-cdk.png" width="150">&nbsp;&nbsp;&nbsp;
 <img alt="SAM" src="docs/img/logo-sam.png" width="55">&nbsp;&nbsp;&nbsp;
 <img alt="Docker Compose" src="docs/img/logo-dockercompose.png" width="80">&nbsp;&nbsp;&nbsp;
+<img alt="Knative" src="docs/img/logo-knative.png" width="80">&nbsp;&nbsp;&nbsp;
+<br>
+<br>
+<img alt="Crossplane" src="docs/img/logo-crossplane.png" width="150">&nbsp;&nbsp;&nbsp;
```

Comment:
I think we should try to choose another img for Crossplane 

<img width="283" alt="image" src="https://user-images.githubusercontent.com/74001161/181715315-bc989a32-98d3-4afa-8106-9f5be37de8e1.png">